setup/install-openroad.sh

```diff
@@ -13,8 +13,16 @@ git checkout $(python3 ${src_path}/_tools.py --tool openroad --field git-commit)
 git submodule update --init --recursive
 
 sudo ./etc/DependencyInstaller.sh
+if [ ! -z ${SC_BUILD} ]; then
+    cp ./etc/DependencyInstaller.sh ${SC_BUILD}/
+fi
```

Comment:
what is purpose of this? I assume Docker related?

Reply:
yeah, I need the DependencyInstaller.sh in a particular location and this ensures it's not in the /deps/ which gets removed immediately after the build.

Reply:
I could remove it for now, since the Docker stuff is still in development

Reply:
I think it's fine, the main thing that stuck out to me was that you're moving it after you call it?

Reply:
trying to keep the docker image small-ish, so after calling `install-openroad.sh && rm -rf /deps` which causes the file to get deleted before I can copy it out